requirements/__init__.py

```diff
@@ -0,0 +1,88 @@
+import os
+import subprocess
+
+from importlib_metadata import metadata
+from importlib_metadata import PackageNotFoundError
+from pip_module_scanner.scanner import Scanner
+
+
+# Override Scanner to skip dotfiles
+class RepoImportScanner(Scanner):
+    """Overwrite Scanner to use git tracked files instead of os.walk
+
+    Also override init to create installed_packages since we just want to write package names
+    """
+
+    def __init__(self, *args, **kwargs):
+        # workaround for https://gitlab.com/python-devs/importlib_metadata/issues/81
+        self.tricky_package_map = kwargs.pop("package_map", None) or {}
+        # overwrite libraries_installed keyed on package names
+        super(RepoImportScanner, self).__init__(*args, **kwargs)
+        self.installed_packages = [
+            lib.key
+            for lib in self.libraries_installed
+            if lib.key not in "manageiq-integration-tests"
+        ]  # ignore local
+
+    def search_script_directory(self, path):
+        """
+        Recursively loop through a directory to find all python
+        script files. When one is found, it is analyzed for import statements
+
+        Only scans files tracked by git
+
+        :param path: string
+        :return: generator
+        """
+        proc = subprocess.Popen(["git", "ls-files", "--full-name"], stdout=subprocess.PIPE)
+        proc.wait()
+        # decode the file names because subprocess PIPE has them as bytes
+        for file in [f.decode() for f in proc.stdout.read().splitlines()]:
```

Comment:
Did you intend to use this keyword? `file`?

Reply:
I blame @mshriver  for that one :grin:

Reply:
Please change it to something non-keyword and give a `--` to @mshriver  only if @jawatts ever enables `--` karma bot

Reply:
I make my own `file`, aight?